pkg/kafka/configkafka/config.go

```diff
@@ -48,6 +48,11 @@ type ClientConfig struct {
 
 	// Metadata holds metadata-related configuration for producers and consumers.
 	Metadata MetadataConfig `mapstructure:"metadata"`
+
+	// Rack provides the rack identifier for this client to enable rack-aware
+	// replica selection when supported by the brokers. This maps to Kafka's
+	// standard "client.rack" setting. By default, this is empty.
+	Rack string `mapstructure:"rack_id"`
```

Comment:
```suggestion
	// RackID provides the rack identifier for this client to enable rack-aware
	// replica selection when supported by the brokers. This maps to Kafka's
	// standard "client.rack" setting. By default, this is empty.
	RackID string `mapstructure:"rack_id"`
```

Let's keep the field name and the JSON/YAML tag name aligned

Reply:
sure. I added your suggestion + changelog